spec/support/capybara.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.configure do |config|
+  config.before :each, type: :system do
+    driven_by :rack_test
```

Comment:
I realise this is kind of redundant but given the work on UI/QA going on I thought it wise to be explicit, but happy to change this.